### PR TITLE
view_home_and_dishes

### PR DESCRIPTION
**Pull request by datToc004**
## Related Tickets:
- [#40315](https://edu-redmine.sun-asterisk.vn/issues/40315)

## WHAT this PR does?

view hompage  and view dishes
## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by datToc004** [comment]
ready

File: resources/views/frontend/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    </div>
+                </a>
+            </div>
+            <div class="col-md-6">
+                <div class="row">
+                    <div class="col-md-6">
+                        <a href="" class="f-product-2"
+                            style="background-image: url(bower_components/start-bower123/images/img-02.jpg);">
+                            <div class="desc">
+                                <h2> <br>Đồ <br> Mới</h2>
```
[the remark this answers is not in the document]
Comment (manhtd-1083) [reply resources/views/frontend/index.blade.php:22]: chưa fix comment

File: resources/views/frontend/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                            </div>
+                        </a>
+                    </div>
+                    <div class="col-md-6">
+                        <a href="" class="f-product-2"
+                            style="background-image: url(bower_components/start-bower123/images/img-03.jpg);">
+                            <div class="desc">
+                                <h2>Sale <br>20% <br>off</h2>
```
Comment (manhtd-1083) [inline resources/views/frontend/index.blade.php:30]: đây nữa nè

File: resources/views/frontend/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                            </div>
+                        </a>
+                    </div>
+                    <div class="col-md-12">
+                        <a href="" class="f-product-2"
+                            style="background-image: url(bower_components/start-bower123/images/img-04.jpg);">
+                            <div class="desc">
+                                <h2>Bánh <br>giảm <br>Cân</h2>
```
Comment (manhtd-1083) [inline resources/views/frontend/index.blade.php:38]: fix hết mấy chỗ ntn nha

**Comment by datToc004** [comment]
ready

**Comment by datToc004** [comment]
ready

**Review by manhtd-1083 — APPROVED** [review]

**Pull request merged**